src/main/java/github/scarsz/discordsrv/objects/managers/link/JdbcAccountLinkManager.java

```diff
@@ -53,7 +53,8 @@
 @SuppressWarnings("SqlResolve")
 public class JdbcAccountLinkManager extends AbstractAccountLinkManager {
 
-    private final static Pattern JDBC_PATTERN = Pattern.compile("([a-z]+)://(.+):(.+)/([A-z0-9]+)"); // https://regex101.com/r/7PSgv6
+    // more accurate RegEx for easier use later on and more accurate matches:
```

Comment:
Added an example of the regex in regex101
```suggestion
    // more accurate RegEx for easier use later on (https://regex101.com/r/sL9BxT/):
```